Updated Windows Install Instructions to Use Latest Recommended Directory Structure

liangfok opened the pull request:
Resolves #3026.

I needed to use Windows to debug a Windows linker error in #3643, and figured I might as well update the instructions as I was following them.

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/robotlocomotion/drake/3652)
<!-- Reviewable:end -->

liangfok commented:
This is a documentation only change that's specific to Windows.

+@sherm1 for both levels of review please.

---

Review status: 0 of 1 files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/3652#-:-KTAbXxhUT91EXWDo7oL:bjs1tgs)*
<!-- Sent from Reviewable.io -->

sherm1 commented:
One minor change needed. Then <img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>.

---

Reviewed 1 of 1 files at r1.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

*[drake/doc/windows.rst, line 175 at r1](https://reviewable.io:443/reviews/robotlocomotion/drake/3652#-KTB7Yw1OfEcNYXgitqg:-KTB7Yw1OfEcNYXgitqh:bw06dzz) ([raw file](https://github.com/robotlocomotion/drake/blob/9c3097e0ac58e9e04717cc488e7e0d7d3173b545/drake/doc/windows.rst#L175)):*
> ```reStructuredText
> 
> * ``drake-distro/build\install\lib``
> * ``drake-distro/build\install\bin``
> ```

Shouldn't mix forward and back slashes. All forward is OK.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/3652#-:-KTB8Qvq7ozONnUh8N8x:b-3dqg9g)*
<!-- Sent from Reviewable.io -->

liangfok commented:
@sherm1, I addressed your comment. Now waiting on CI.

---

Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion, some commit checks pending.

---

*[drake/doc/windows.rst, line 175 at r1](https://reviewable.io:443/reviews/robotlocomotion/drake/3652#-KTB7Yw1OfEcNYXgitqg:-KTB9qc9GRM8wvAkfQGz:b-896fix) ([raw file](https://github.com/robotlocomotion/drake/blob/9c3097e0ac58e9e04717cc488e7e0d7d3173b545/drake/doc/windows.rst#L175)):*
<details><summary><i>Previously, sherm1 (Michael Sherman) wrote…</i></summary>
> Shouldn't mix forward and back slashes. All forward is OK.
</details>
Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/3652#-:-KTB9tL1i-Jr408TxMZy:b-rjp3tr)*
<!-- Sent from Reviewable.io -->
